packages/usa-combo-box/src/styles/_usa-combo-box.scss

```diff
@@ -110,8 +129,9 @@ button.usa-combo-box__clear-input {
   background-color: color("gray-cool-20");
   position: absolute;
   top: 1px;
-  margin-bottom: 8px;
-  margin-top: 8px;
+  height: calc(100% - #{units(2)});
+  margin-bottom: units(1);
+  margin-top: units(1);
```

Comment:
The separator was not vertically aligned, so I matched the margin to what is declared in `%block-input-styles` and extended it to be the full height